notebooks/using-mlflow/train-with-mlflow/mlflow-v1-comparison.ipynb

```diff
@@ -884,9 +884,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.8 - AzureML",
+   "display_name": "Python (heart)",
```

Comment:
this need to be changed by to ""Python 3.8 - AzureML", don't want users having to install a new kernel is they dont have to

Reply:
This is not a significant deal cause we never know which kernel the user is using. When a notebook indicates a kernel the user doesn't have, it prompts "which kernel do you want to use?". "Python 3.8 - AzureML" is the one we have at Compute Instances, but if the user runs it locally it will have the same scenario.  Does it make sense?